### ISSUE TITLE
definition improvement for tactileObject

### ISSUE THREAD

**Issue by clapierre** [w3c/a11y-discov-vocab#16]
4.2.4.13 tactileObject

The content is a tactile 3D object, or the model to generate one is included.

I would think this should be
The content contains a tactile 3D object, or the model to generate one is included.

**Comment by madeleinerothberg** [comment]
The use case for the first half of the sentence is, I think, that metadata is being used to describe a physical object filed in a drawer or shelf somewhere. And the second half is for describing a digital object that tells you how to make a physical object. 

I'm not sure how changing "is" to "contains" changes this but if others feel it is clearer, I don't object.

**Comment by clapierre** [comment]
Just wanted to make sure it was clear that the entire EPUB doesn't need to be  tactile objects, but if some appendix is a tactile object then having this metadata is valid.

**Comment by mattgarrish** [comment]
Can a tactile object have an accessibility feature of a tactile object?

When we're describing a physical tactile object or graphic, I keep thinking that TactileGraphic and TactileObject should be subtypes of CreativeWork so you can talk about them externally to this metadata.

**Comment by clapierre** [comment]
I agree that a TactileGraphic / TactileObject should be subtypes of CreativeWork that makes sense.
I guess all this metadata implies that in this EPUB there will be at least one occurrence of  XYZ within.

However now this would mean going to schema.org and asking that these be subtypes of CreativeWorks right?  That will take a while :). I think we could still include them in our metadata under accessibilityFeature not sure how that would work to point to another schema.org property like derivative of a CreativeWork from accessibilityFeature.

**Comment by madeleinerothberg** [comment]
Matt asks: "Can a tactile object have an accessibility feature of a tactile object?" I don't see why not. A fixed-size large print book can have the large print accessibilityFeature. A braille file has accessilityFeature braille.

I agree that ultimately it would make sense to have a CreativeWork property for objects that are only a tactile. But this vocab works in the short term and also covers the continuing case of a digital tactile recipe included within a larger digital object such as an EPUB or Web site.

**Comment by mattgarrish** [comment]
> A fixed-size large print book can have the large print accessibilityFeature.

Right, but what I was trying to get at is that in this scenario we're saying that a creative work/book has a feature of being formatted for large print reading. It's not true of all books, so it's a feature.

Standalone tactile graphics and objects, on the other hand, are unique "things", like books, or that's at least my perception of them. It sounds strange to describe their inherent characteristic in terms of a feature.

I doubt we'll get more types into schema.org unless we can show they're being used, though, so making a case for standalone tactile graphics and objects is probably not worth the effort. I can live with them as features, but, like @clapierre, I think of the wording more along the lines of a book "containing" the graphics.

**Comment by madeleinerothberg** [comment]
Ok, sounds fine.
